internal/user/file.go

```diff
@@ -0,0 +1,49 @@
+package user
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/tilt-dev/wmclient/pkg/dirs"
+	"gopkg.in/yaml.v3"
+)
+
+const userPrefsFileName = "tilt_user_prefs.yaml"
+
+type filePrefs struct {
+	dir *dirs.TiltDevDir
+}
+
+func NewFilePrefsInterface(dir *dirs.TiltDevDir) *filePrefs {
```

Comment:
feels even weirder to call this `New...Interface` when the thing it's returning is just a struct. `NewFilePrefs`?

Reply:
done